src/DashboardHub/Bundle/AppBundle/Entity/User.php

```diff
@@ -35,6 +35,17 @@ public function getId()
     {
         return $this->id;
     }
+    
+    /**
+     * Return a password to be used by the development bundle
+     * as opposed to the null returned by the parent class
+     * 
+     * @return string
+     */
+    public function getPassword()
+    {
+        return 'IAmNotASecret';
```

Comment:
I have moved this to a **MockUser** in the `DevelopmentBundle` that inherits from this class. Seems to be working fine.